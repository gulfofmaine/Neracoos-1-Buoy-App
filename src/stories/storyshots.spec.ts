import initStoryshots from "@storybook/addon-storyshots"

// Mock next/navigation hooks for all storyshots
jest.mock("next/navigation", () => {
  const origionalModule = jest.requireActual("next/navigation")

  return {
    __esModule: true,
    ...origionalModule,
    usePathname: jest.fn(() => "/"),
  }
})

initStoryshots({})
