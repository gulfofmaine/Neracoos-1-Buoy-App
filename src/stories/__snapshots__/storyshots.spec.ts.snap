// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/500 Page Five Hundred Page 1`] = `
<div
  className="row"
>
  <div
    className="col-md-8 offset-md-2"
  >
    <h1>
      The data has escaped
    </h1>
    <img
      src="n_walkabout.png"
      style={
        Object {
          "maxWidth": "100%",
        }
      }
    />
    <p>
      Despite our best intentions, something has gone wrong while trying to display the data.
    </p>
    <p>
      If you saw a 
      <b>
        It looks like we're having issues
      </b>
       form, we hope that you filled it out, as it will help us fix the issue. Otherwise you can email 
      <a
        href="mailto:info@neracoos.org"
      >
        info@neracoos.org
      </a>
       with information about the error.
    </p>
    <p>
      Sometimes issues are short lived, so if you give it a minute, then try reloading the page, hopefully you will be able to get to the data. If you can't and this error shows up again, please fill out the form, or email us for help.
    </p>
  </div>
</div>
`;

exports[`Storyshots Components/Footer Footer 1`] = `
<div
  className="footer"
>
  <div
    className="row"
    style={
      Object {
        "fontSize": ".8rem",
        "padding": ".25rem",
        "paddingTop": "1rem",
      }
    }
  >
    <div
      className="mx-auto text-center col-md-10"
    >
      Copyright Â© 2021 ~ NERACOOS
    </div>
  </div>
  <div
    className="row"
    style={
      Object {
        "fontSize": ".8rem",
        "padding": ".25rem",
      }
    }
  >
    <div
      className="mx-auto text-center col-md-10"
    >
      Use
       
      <a
        href="https://docs.google.com/forms/d/e/1FAIpQLSdzP90655d-ZuDGpdgcMmTvxW1sgR_Hg4KI1KCfQDFU8MMF0g/viewform?usp=sf_link"
      >
        this form
      </a>
       
      to send us feedback or bug reports
    </div>
  </div>
  <div
    className="row"
    style={
      Object {
        "fontSize": ".8rem",
        "padding": ".25rem",
      }
    }
  >
    <div
      className="mx-auto text-center col-md-10"
    >
      Product of 
      <a
        href="http://www.neracoos.org/"
      >
        NERACOOS.org
      </a>
       - Developed and maintained by the
       
      <a
        href="https://gmri.org/commitments/science/ocean-data-products/"
      >
        Gulf of Maine Research Institute
      </a>
    </div>
  </div>
</div>
`;

exports[`Storyshots Components/NavBar Nav Bar 1`] = `
<div>
  <nav
    className="navbar navbar-expand-md navbar-dark"
  >
    <div
      className="container-fluid"
    >
      <a
        className="navbar-brand"
        href="http://neracoos.org"
      >
        <img
          alt="NERACOOS"
          src="neracoos_logo.png"
        />
      </a>
      <button
        aria-label="Toggle navigation"
        className="navbar-toggler"
        onClick={[Function]}
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="justify-content-end collapse navbar-collapse"
        style={Object {}}
      >
        <ul
          className="ml-auto navbar-nav"
        >
          <a
            aria-current="page"
            className="nav-link active"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
          <li
            className="dropdown nav-item"
            onKeyDown={[Function]}
          >
            <a
              aria-expanded={false}
              aria-haspopup={true}
              className="dropdown-toggle nav-link"
              href="#"
              onClick={[Function]}
            >
              Regions
            </a>
            <div
              aria-hidden={true}
              className="dropdown-menu dropdown-menu-end"
              role="menu"
              tabIndex="-1"
            >
              <a
                className="nav-link dropdown-item btn"
                href="/platform/?region=GOM"
                onClick={[Function]}
              >
                Gulf Of Maine
              </a>
              <a
                className="nav-link dropdown-item btn"
                href="/platform/?region=LONG"
                onClick={[Function]}
              >
                Long Island Sound
              </a>
              <a
                className="nav-link dropdown-item btn"
                href="/platform/?region=GREAT"
                onClick={[Function]}
              >
                Great Bay, NH
              </a>
              <a
                className="nav-link dropdown-item btn"
                href="/platform/?region=BOSTON"
                onClick={[Function]}
              >
                Boston Harbor
              </a>
              <a
                className="nav-link dropdown-item btn"
                href="/platform/?region=CAPE"
                onClick={[Function]}
              >
                Cape Cod / Buzzards Bay
              </a>
              <a
                className="nav-link dropdown-item btn"
                href="/platform/?region=NARRAGANSETT"
                onClick={[Function]}
              >
                Narragansett Bay
              </a>
              <a
                className="nav-link dropdown-item btn"
                href="/platform/?region=NEWFOUNDLAND"
                onClick={[Function]}
              >
                Newfoundland
              </a>
            </div>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/models/"
              onClick={[Function]}
            >
              Model Viewer
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="/about"
              onClick={[Function]}
            >
              About
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
</div>
`;

exports[`Storyshots ERDDAP/Info/Panel Platform Info 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <div
      className="card-title"
    >
      Station 
      M01
    </div>
    <p
      className="card-text"
    >
      Jordan Basin
      <br />
      <b>
        Lat:
      </b>
       
      43.5
       
      <b>
        Lon:
      </b>
       
      -67.87
      <br />
      <a
        href="http://gyre.umeoce.maine.edu/gomoos.php"
      >
        University of Maine
      </a>
      <br />
    </p>
  </div>
</div>
`;

exports[`Storyshots ERDDAP/Superlatives Configureable 1`] = `
<div
  className="card"
  style={
    Object {
      "marginBottom": "1rem",
      "marginTop": "1rem",
    }
  }
>
  <div
    className="card-header"
  >
    <h5>
      Latest Conditions
    </h5>
  </div>
  <div
    className="card-body"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <a
          href="/platform/WLIS"
          onClick={[Function]}
        >
          <h6>
            Highest Winds
          </h6>
        </a>
        <div>
          22.8
           
          Knots
        </div>
        <a
          href="/platform/WLIS"
          onClick={[Function]}
        >
          <div>
            WLIS
             - 
            Western Long Island Sound
          </div>
        </a>
      </div>
      <div
        className="col"
      >
        <a
          href="/platform/44008"
          onClick={[Function]}
        >
          <h6>
            Biggest Waves
          </h6>
        </a>
        <div>
          8.3
           
          Feet
        </div>
        <a
          href="/platform/44008"
          onClick={[Function]}
        >
          <div>
            44008
             - 
            Nantucket 54NM Southeast of Nantucket
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots ERDDAP/Superlatives Should Not Show Outdated Data 1`] = `null`;

exports[`Storyshots Units/UnitSelector/configurable English 1`] = `
<div
  className="unit-selector btn-group"
  role="group"
>
  <button
    aria-label={null}
    className="btn btn-light btn-sm"
    id="Metric"
    onClick={[Function]}
    type="button"
  >
    Metric
  </button>
  <button
    aria-label={null}
    className="btn btn-primary btn-sm active"
    id="English"
    onClick={[Function]}
    type="button"
  >
    English
  </button>
</div>
`;

exports[`Storyshots Units/UnitSelector/configurable Metric 1`] = `
<div
  className="unit-selector btn-group"
  role="group"
>
  <button
    aria-label={null}
    className="btn btn-primary btn-sm active"
    id="Metric"
    onClick={[Function]}
    type="button"
  >
    Metric
  </button>
  <button
    aria-label={null}
    className="btn btn-light btn-sm"
    id="English"
    onClick={[Function]}
    type="button"
  >
    English
  </button>
</div>
`;
